tornjak-frontend/src/components/tornjak-helper.js

```diff
@@ -1,5 +1,5 @@
 // Additional objects the functions take
-// entriesMetaData, agentsSpiffeIdDict, WorkLoadAttestorInfo, entryClusterMetadata
+// entriesMetaData, agentsSpiffeIdDict, workLoadAttestorInfo, clusterEntry
```

Comment:
can you help define what some of these look like? It should point to some documentation

Reply:
done!